src/components/atoms/Navigation/NavMobile.tsx

```diff
@@ -122,9 +28,7 @@ const NavMobile: React.FC<NavMobileProps> = ({
           <ButtonClose onClick={onClickClose} />
         </span>
       </div>
-      <ul className="flex flex-col py-6 px-2 space-y-1">
-        {data.map(_renderItem)}
-      </ul>
+      <ul className="flex flex-col py-6 px-2 space-y-1"></ul>
```

Comment:
Why `{data.map(_renderItem)}` was removed?

Reply:
Will we still need this line?

Reply:
we are not using in the application. Its a fake data with products variants

Reply:
so, will we need the <ul> tag for what? why we dont remove it also?

Reply:
thats my mistake. i forgot to remove the html tag